cluster-provision/k8s/1.22-ipv6/provision.sh

```diff
@@ -94,7 +94,7 @@ export PATH=$ISTIO_BIN_DIR:$PATH
 # generate Istio manifests for pre-pulling images
 istioctl manifest generate --set profile=demo --set components.cni.enabled=true | tee /tmp/istio-deployment.yaml
 
-export CRIO_VERSION=1.20
+export CRIO_VERSION=1.21
```

Comment:
It is recommended to match crio with kubernetes version.

Reply:
Yes, but we don't heve 1.22 mirrored yet. I could do this only for 1.21 for now and post another PR for 1.22 when It's done

Reply:
I see. I thought we don't run provisioning that often.

Reply:
Keep in mind we already have 1.23 :)

Reply:
Maybe we can switch to 1.22 now as the mirror pr was merged. WDYT?

Reply:
qq: who is handling the mirroring?

Reply:
fyi https://github.com/kubevirt/project-infra/pull/1878

Reply:
> Maybe we can switch to 1.22 now as the mirror pr was merged. WDYT?

we had to wait for the sync to be triggered, which is at Sunday night.
Unfortunately, it failed. https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/periodic-kubevirtci-mirror-crio-repository-weekly/1487945070802898944

@dhiller This is the periodic job for syncing the repos https://github.com/kubevirt/project-infra/blob/2ffb5944e61b4b84068fc54613f175d7fd2f4d12/github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml#L2

Reply:
> > Maybe we can switch to 1.22 now as the mirror pr was merged. WDYT?
> 
> we had to wait for the sync to be triggered, which is at Sunday night. Unfortunately, it failed. https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/periodic-kubevirtci-mirror-crio-repository-weekly/1487945070802898944
>

You can ping me, I can trigger the job manually if needed.

Reply:
Thank you @dhiller, it's already synced